src/test/java/org/testeditor/fixture/web/FirefoxTest.java

```diff
@@ -86,7 +86,36 @@ public void googleTestWithSpecialKeys() throws FixtureException {
         String title = driver.getTitle();
         Assert.assertTrue(title.startsWith("Edito - Google"));
     }
+    
+    @Test
+    public void testWithSpecialInputFields() throws FixtureException {
+        // given
+        String searchField = "q";
+        driver.startBrowser("firefox");
+        driver.goToUrl("https://google.de");
+        driver.typeInto(searchField, LocatorStrategy.NAME, "Test-Editor");
+       
+        // when
+        String readValue = driver.readValue(searchField, LocatorStrategy.NAME);
```

Comment:
does this test the static or the dynamic case?

Reply:
This test case tests the dynamic part -> first type something into the inputfield and verify if this is available in this field. I added Unit tests for this cases.

Reply:
I added unit tests which tests both variants ;O)